Remove python detection from configure.in, we won't be installing the gdb python
scripts anymore. They use a redhat-specific gdb module that has not been
upstreamed yet.

https://bugs.gentoo.org/291328
https://bugzilla.gnome.org/623552
---
--- configure.in
+++ configure.in
@@ -379,10 +379,6 @@ if test "x$PERL_PATH" = x ; then
 fi
 AC_SUBST(PERL_PATH)
 
-# Need suitable python path for greport
-AM_PATH_PYTHON(2.4,,PYTHON="/usr/bin/env python2.4")
-
-
 dnl ***********************
 dnl *** Tests for iconv ***
 dnl ***********************
